test/inductor/test_cpp_wrapper_hipify.py

```diff
@@ -36,16 +36,16 @@ def test_hipify_basic_declaration(self) -> None:
     def test_hipify_aoti_driver_header(self) -> None:
         header = cuda_kernel_driver()
         expected = """
-            #define CUDA_DRIVER_CHECK(EXPR)                     \\
-            do {                                                \\
-                hipError_t code = EXPR;                         \\
-                const char *msg;                                \\
-                std::ignore = hipDrvGetErrorString(code, &msg); \\
-                if (code != hipSuccess) {                       \\
-                    throw std::runtime_error(                   \\
-                        std::string("CUDA driver error: ") +    \\
-                        std::string(msg));                      \\
-                }                                               \\
+            #define CUDA_DRIVER_CHECK(EXPR)                    \\
```

Comment:
NIT, not sure why the indentation changed? 
Since this is temp change its ok.

Reply:
@pruthvistony Indentation needs to be like this because it's straight in the original code in `cuda_codegen.kernel_driver()`. After hipification it gets shifted and we need to check for this version since the check is very precise with string equality.